tracing-appender/src/rolling/builder.rs

```diff
@@ -38,16 +39,21 @@ impl Builder {
     /// | Parameter | Default Value | Notes |
     /// | :-------- | :------------ | :---- |
     /// | [`rotation`] | [`Rotation::NEVER`] | By default, log files will never be rotated. |
-    /// | [`filename_prefix`] | `""` | By default, log file names will not have a prefix. |
+    /// | [`prefix`] | `""` | By default, log file names will not have a prefix. |
+    /// | [`suffix`] | `""` | By default, log file names will not have a suffix. |
+    /// | [`max_files`] | `None` | By default, there is no limit for maximum log file count. |
```

Comment:
looks like the [docs build is failing](https://app.netlify.com/sites/tracing-rs/deploys/632f69c21714ec000916a8d9#L241-L262) because we're not using the actual names of the methods, but the names of the fields on the builder (which are private). let's change this to:
```suggestion
    /// | [`filename_prefix`] | `""` | By default, log file names will not have a prefix. |
    /// | [`filename_suffix`] | `""` | By default, log file names will not have a suffix. |
    /// | [`max_log_files`] | `None` | By default, there is no limit for maximum log file count. |
```
so that the links actually work?

Reply:
oh, looks like you already fixed it while i was leaving the comment --- thanks!

Reply:
no they didn't, I erroneously assumed they were referring to the parameters, not the functions. Fixed just now after checking netlify logs